flags.go

```diff
@@ -76,9 +76,9 @@ func FlagUsages(f *pflag.FlagSet) string {
 		line += "\n    - " + usage
 		if !defaultIsZeroValue(flag) {
 			if flag.Value.Type() == stringType {
-				line += fmt.Sprintf(" (default %q)", flag.DefValue)
+				line += fmt.Sprintf(" This value defaults to %q", flag.DefValue)
```

Comment:
@JuliaMongo should this be 
```suggestion
				line += fmt.Sprintf(". This option defaults to %q", flag.DefValue)
```

to make it two sentences? also optional, changing value for option

Reply:
all the flag descriptions end with a full stop https://github.com/mongodb/mongocli/blob/master/internal/usage/usage.go#L18

Reply:
If we do this, we will end with some usage with `.` and some without it